src/Controller/PanelsController.php

```diff
@@ -53,11 +53,9 @@ public function beforeFilter(Event $event)
      */
     public function beforeRender(Event $event)
     {
-        $builder = $this->viewBuilder();
-        if (!$builder->className()) {
-            $builder->layout('DebugKit.panel')
-                ->className('DebugKit.Ajax');
-        }
+        $this->viewBuilder()
+            ->layout('DebugKit.panel')
+            ->className('DebugKit.Ajax');
```

Comment:
This will break fetching the history data. I tested locally and it results in 500 errors containing `"Template file "Panels/index.ctp" is missing."`